create-cycle-app/index.js

```diff
@@ -70,34 +88,51 @@ function createApp (name, verbose, flavor) {
       preparePackageJson(appFolder, appName, flavor, answers.streamLib, verbose)
     })
   } else {
-    fetchFlavors(function (err, flavors) {
-      if (err) {
-        throw err
-      }
+    inquirer.prompt([
+      {
+        name: 'flavor',
+        message: 'Which flavor do you want to use?',
+        type: 'list',
+        choices: coreFlavors.sort(byName).concat(discoverMore)
+      },
+      streamLibQuestion
+    ]).then(function (answers) {
+      // When run-discovery was choosed, we need to discover
+      // flavors somewhere else
+      if (answers.flavor === 'run-discovery') {
+        discoverFlavors(function (err, flavors) {
+          if (err) {
+            throw err
+          }
 
-      inquirer.prompt([
-        {
-          name: 'flavor',
-          message: 'Which flavor do you want to use?',
-          type: 'list',
-          choices: flavors
-        },
-        streamLibQuestion
-      ]).then(function (answers) {
+          inquirer.prompt([
+            {
+              name: 'flavor',
+              message: 'Which flavor do you want to use?',
+              type: 'list',
+              choices: flavors.sort(byName).concat(coreFlavors.filter(withoutDiscovery))
+            },
+            streamLibQuestion
+          ]).then(function (answers) {
+            preparePackageJson(appFolder, appName, answers.flavor, answers.streamLib, verbose)
+          })
+        })
+      } else {
         preparePackageJson(appFolder, appName, answers.flavor, answers.streamLib, verbose)
-      })
+      }
     })
   }
 }
 
-function fetchFlavors (cb) {
+function discoverFlavors (cb) {
   console.log()
-  console.log(chalk.cyan('Fetching available flavors'))
+  console.log(chalk.cyan('Fetching community flavors...'))
   console.log()
 
-  // NOTE: Maybe change the method to discover flavors
+  // NOTE: change the method to discover flavors to the published communityFlavors.json
+  // following is just for testing
   request({
-    url: 'https://api.github.com/gists/0f33b55f62baca22c6bdb73b56333311',
+    url: 'https://api.github.com/repos/nickbalestra/create-cycle-app-community-flavors/contents/communityFlavors.json',
```

Comment:
yup that was just for testing purpose, it should probably point to: `https://api.github.com/repos/cyclejs-community/create-cycle-app/contents/communityFlavors.json`